scalyr_agent/configuration.py

```diff
@@ -540,6 +540,10 @@ def disable_bandwidth_stats(self):
     def disable_update_debug_log_level(self):
         return self.__get_config().get_bool("disable_leak_update_debug_log_level")
 
+    @property
+    def enable_gc_stats(self):
+        return self.__get_config().get_bool("enable_gc_stats")
```

Comment:
Can you add an entry to the test cases in `configuration_test.py` for each top-level configuration option.  We typically add one to `TestConfiguration.test_basic_case` to verify the default is set right.. and then another to  `TestConfiguration.test_overriding_basic_settings` to verify we can override.  I just like to have a double-check on the config names and their default values since once customers use them, it's hard to change them.

Reply:
Yep, I plan to add some tests and hook new benchmark job to Circle CI :)